src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java

```diff
@@ -90,6 +79,28 @@ public void mercury_navigation_navigationOnNonEnglishWiki() {
 
     topBar.openNavigation();
     Assertion.assertFalse(navigation.areHubLinksVisible());
-    Assertion.assertFalse(navigation.isFooterVisible());
   }
+
+  @Execute(asUser = User.USER)
+  @Test(groups = "mercury_navigation_navigationElementsUserLoggedIn")
+  public void mercury_navigation_navigationElementsUserLoggedIn() {
+    init();
+
+    topBar.openNavigation();
+    Assertion.assertTrue(navigation.isUserAvatarVisible());
+    Assertion.assertTrue(navigation.isUserProfileLinkVisible());
+    Assertion.assertTrue(navigation.isLogoutLinkVisible());
+  }
+
+  @Execute(asUser = User.ANONYMOUS)
+  @Test(groups = "mercury_navigation_navigationElementsAnonymousUser")
+  public void mercury_navigation_navigationElementsAnonymousUser() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)